sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpClientTests.cs

```diff
@@ -300,7 +300,7 @@ public void GetPropertiesAsyncAppliesTheRetryPolicyForAmqpErrors(EventHubsRetryO
             var eventHubName = "myName";
             var tokenValue = "123ABC";
             var retryPolicy = new BasicRetryPolicy(retryOptions);
-            var retriableException = AmqpError.CreateExceptionForError(new Error { Value = AmqpError.ServerBusyError }, "dummy");
+            var retriableException = AmqpError.CreateExceptionForError(new Error { Condition = AmqpError.ServerBusyError }, "dummy");
```

Comment:
These fixes are orthogonal, but were discovered during parallel efforts on Service Bus.  While the test outcome is not impacted, `Conditional` is the correct property to be assigned.